### PR TITLE
mod_ssl: Added documentation on important ssl app settings

### PR DESCRIPTION
**Pull request by mmzeeman**
PR as discussed in #1335

### CODE REVIEW

File: priv/erlang.config.in [hunk 1]
```diff
@@ -38,6 +38,15 @@
    {crash_log, "priv/log/crash.log"}
   ]},
 
+ {ssl, 
+  [
+%%% The number of maximum number of ssl sessions cached by the server. Increase if you 
+%%% of incoming connections.
```
Comment (arjan) [inline priv/erlang.config.in]: "Increase if you *run out of* incoming connections."?
￼

**Comment by mmzeeman** [comment]
Fails because of an xref issue with mochijson2

File: priv/erlang.config.in (hunk 1 above)
Comment (mmzeeman) [inline priv/erlang.config.in]: You can't run out of them, it is just a cache with session data. When the max is reached it will invalidate all entries.

Comment (mmzeeman) [inline priv/erlang.config.in]: It contained "a lot of"

**Comment by ddeboer** [comment]
Fixing the build in #1338.

**Comment by ddeboer** [comment]
Build fixed on master. Please rebase this onto that.

**Comment by mmzeeman** [comment]
I don't have a computer right now. Later this week maybe.

**Comment by mworrell** [comment]
👍 🌴 📚

**Pull request merged**